movingSquare.js

```diff
@@ -0,0 +1,93 @@
+const square = document.createElement('div')
+const h6 = document.createElement('h6')
+
+let positionX = 0
+let positionY = 0
+
+square.style.backgroundColor = 'red'
+square.style.width = '100px'
+square.style.height = '100px'
+square.style.position = 'absolute'
+square.style.left = positionX + 'px'
+square.style.top = positionY + 'px'
+square.style.zIndex = 1
+
+h6.innerHTML = 'X=' + String(positionX) + ' | ' + 'Y=' + String(positionY)
```

Comment:
Numbers are automatically converted to strings when one of the variables around + operator is a string. The best way though is to use template strings here.